.tekton/pull-request.yaml

```diff
@@ -95,6 +95,20 @@ spec:
               image: registry.access.redhat.com/ubi8/ubi:latest
               workingDir: $(workspaces.workspace.path)
               script: yum install -y make && make pint_lint
+      - name: lint-prometheus-rules-yaml-lint
+        workspaces:
+          - name: workspace
+            workspace: workspace
+        runAfter:
+          - prepare-for-prometheus-tests
```

Comment:
It doesn't have to run after `prepare-for-prometheus-tests` 
Actually, I would exclude the Prometheus rules files as they are already linted by promtools and pint.

Reply:
> It doesn't have to run after `prepare-for-prometheus-tests` Actually, I would exclude the Prometheus rules files as they are already linted by promtools and pint and the extraction of the rules is not relevant for yamllint (promtools and pint can't run without the extraction though)

Excluded those lines. Thanks